stock_picking_yamato_csv/tests/test_stock_picking_yamato_csv.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Quartile Limited
+
+from odoo.exceptions import UserError
+from odoo.tests import SavepointCase, tagged
```

Comment:
```suggestion
from odoo.tests.common import SavepointCase
```

Reply:
@yostashiro 
Test doesn't run caused by depend structure, so this time I will not test.